scripts/enable_automatic_updates.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+apt-get install unattended-upgrades curl
+
+echo '
+APT::Periodic::Update-Package-Lists "1";
+APT::Periodic::Unattended-Upgrade "1";
+' >/etc/apt/apt.conf.d/20auto-upgrades
+
+sed -i.bak 's|//Unattended-Upgrade::Remove-Unused-Dependencies "false"|Unattended-Upgrade::Remove-Unused-Dependencies "true"|' /etc/apt/apt.conf.d/50unattended-upgrades
+
+sed -i.bak 's|//Unattended-Upgrade::Automatic-Reboot "false"|Unattended-Upgrade::Automatic-Reboot "false"|' /etc/apt/apt.conf.d/50unattended-upgrades
```

Comment:
This line doesn't do anything, right? I assume you've put it here to make it clear how to enable automatic reboot in the future. It might be clearer to replace it with a line that actually does enable automatic reboot, and then comment it out?

Reply:
I see. Maybe add a comment to these two lines, as it's quite easy to miss? Or maybe I'm just not paying enough attention.

Reply:
Comment added - yes, easy to miss, the "//" or assume it's a syntactic part of the s|//